exercise_perseus_render/assets/src/vue/attemptprogress.vue

```diff
@@ -9,9 +9,9 @@
       </div>
     </div>
 
-    <p id="message" v-if="passRatioN === passNum">{{ $tr("get") }} <b>{{passRatioN}} {{ $tr("outof") }} {{passRatioM}}</b> {{ $tr("correct") }}</p>
-    <p id="message" v-if="passNum > 0 && passRatioN !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
-    <p id="message" v-if="passNum <= 0 && passRatioN !== passNum">{{ $tr("hooray") }}</p>
+    <p id="message" v-if="passRatioM === passNum">{{ $tr("get") }} <b>{{passRatioM}} {{ $tr("outof") }} {{passRatioN}}</b> {{ $tr("correct") }}</p>
+    <p id="message" v-if="passNum > 0 && passRatioM !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
+    <p id="message" v-if="passNum <= 0 && passRatioM !== passNum">{{ $tr("hooray") }}</p>
```

Comment:
That may be true, but we don't want people on crowdin to translate it like that because they'll have to repeat the work when we switch to proper ICU message syntax.

We should take out the `$tr` functions here to avoid making life hard for any potential translator.